routes/data.js

```diff
@@ -43,12 +43,17 @@ exports.getAllOrganizations = async function () {
 
   const parsedOrganizations = [];
 
-  organizations.docs.forEach((doc) => {
-    const data = doc.data();
-    data['id'] = doc.id;
+  for (const key in organizations.docs) {
+    if (Object.prototype.hasOwnProperty.call(organizations.docs, key)) {
```

Comment:
1. What is ```key``` here? An organization?
2. Can we do ```organizations.docs.hasOwnProperty(key)```?

Reply:
Key is the document id; I changed it to `id` for clarity.

The linter gives the error `Do not access Object.prototype method 'hasOwnProperty' from target object` for `organizations.docs.hasOwnProperty(key)`. I used `Object.prototype.hasOwnProperty.call(organizations.docs, key)` to avoid it.